### PR TITLE
GroupUnderdogs-maksin-refactoring TF

### PR DESCRIPTION
**Pull request by maksin82**

### CODE REVIEW

**Review by nikabenz — APPROVED** [review]

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 1]
```diff
@@ -61,15 +61,14 @@ public void tereshenkov99BottlesLastMenuLinkTestGetText() {
     }
 
 
-    //TC_11_07 что надпись об исключении красным цветом и с маленькой буквы
+    // надпись об исключении красным цветом и с маленькой буквы
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:64]: Let's write comments in English

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 2]
```diff
@@ -61,15 +61,14 @@ public void tereshenkov99BottlesLastMenuLinkTestGetText() {
...
     @Test
     public void maksinTestInactive() {
         driver = new FirefoxDriver();
         driver.get("http://www.99-bottles-of-beer.net/team.html");
         WebElement header = driver.findElement(By.xpath
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:69]: WebElement _header_


is it really header?

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 3]
```diff
@@ -61,15 +61,14 @@ public void tereshenkov99BottlesLastMenuLinkTestGetText() {
...
                 ("/html/body/div/div[3]/p[7]/font/b"));
-        //Assert.assertTrue(header.getText().toLowerCase().equals(header.getText())); //одинаковые
-        Assert.assertEquals(header.getText(), header.getText().toLowerCase());     //одинаковые
+        Assert.assertEquals(header.getText(), header.getText().toLowerCase());
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:71]: Do we really need this line?

Check that the text that is written in lower case equals the same text that is forced to be written in lower case - I guess the result of the check is obvious and always true

**Review by SafronovaOlga — APPROVED** [review]

**Comment by tereshenkov29** [comment]
@SafronovaOlga could you possibly check my comments before approving the request? I believe the code should be updated

**Pull request closed without merging**